rpc/jsonrpc/daemon.go

```diff
@@ -95,12 +108,21 @@ func APIList(db kv.TemporalRoDB, eth rpchelper.ApiBackend, txPool txpool.TxpoolC
 				Version:   "1.0",
 			})
 		case "net":
-			list = append(list, rpc.API{
-				Namespace: "net",
-				Public:    true,
-				Service:   NetAPI(netImpl),
-				Version:   "1.0",
-			})
+			if !isArbitrum {
+				list = append(list, rpc.API{
+					Namespace: "net",
+					Public:    true,
+					Service:   NetAPI(netImpl),
+					Version:   "1.0",
+				})
+			} else {
```

Comment:
seeking for a suggestion here, to be fully compatible (both message and error code) it is needed to provide different interface with disable methods being undeclared. For smaller apis like `net` not big change needed, but for `eth` group it is needed to change 10-20 files

or can manually check if we are on arbitrum inside each method and return error message, but then error code will be different `"code": -32600` instead of `"code": -32601`